HW_8/view.py

```diff
@@ -0,0 +1,53 @@
+from logger import log
+
+
+@log
+def greetings():
+    print('Список персонала')
+
+
+@log
+def show_menu():
+    while True:
+        try:
+            value = int(input(
+                '''\nВыберите команду:\n1 - Просмотреть весь список сотрудников.\n2 - Добавить сотрудника.\n3 - Изменить данные сотрудника.\n4 - Удалить сотрудника.\n0 - Выход.\n'''))
+            if value > 4:
+                print('Введите число от 0 до 4')
+                continue
+            elif value == 0:
+                exit()
```

Comment:
Не помню, писал я вам уже или нет, но таким образом лучше программу не завершать. Но если прям очень хочется, то лучше делать так
```suggestion
                sys.exit()
```

Reply:
Прерывать программу в этом модуле не совсем правильно с точки зрения архитектуры. К тому же в контроллере у вас уже есть соответствующие инструкция.